src/main/java/bio/terra/service/filedata/google/gcs/GcsBufferedReader.java

```diff
@@ -1,16 +1,25 @@
 package bio.terra.service.filedata.google.gcs;
 
+import com.google.cloud.storage.Blob;
+import com.google.cloud.storage.BlobInfo;
 import com.google.cloud.storage.Storage;
 import java.io.BufferedReader;
 import java.nio.channels.Channels;
 import java.nio.charset.StandardCharsets;
 
 /** Given a gs path and a storage object, return a buffered reader for the blob */
 public class GcsBufferedReader extends BufferedReader {
```

Comment:
`BlobReadChannel` is buffered internally so I don't it's recommended to wrap it in a `BufferedReader`.